geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ExportDataCommand.java

```diff
@@ -41,44 +44,34 @@ public Result exportData(
       @CliOption(key = CliStrings.EXPORT_DATA__REGION, mandatory = true,
           optionContext = ConverterHint.REGION_PATH,
           help = CliStrings.EXPORT_DATA__REGION__HELP) String regionName,
-      @CliOption(key = CliStrings.EXPORT_DATA__FILE, mandatory = true,
+      @CliOption(key = CliStrings.EXPORT_DATA__FILE,
           help = CliStrings.EXPORT_DATA__FILE__HELP) String filePath,
+      @CliOption(key = CliStrings.EXPORT_DATA__DIR,
+          help = CliStrings.EXPORT_DATA__DIR__HELP) String dirPath,
       @CliOption(key = CliStrings.MEMBER, optionContext = ConverterHint.MEMBERIDNAME,
-          mandatory = true, help = CliStrings.EXPORT_DATA__MEMBER__HELP) String memberNameOrId) {
+          mandatory = true, help = CliStrings.EXPORT_DATA__MEMBER__HELP) String memberNameOrId,
+      @CliOption(key = CliStrings.EXPORT_DATA__PARALLEL, unspecifiedDefaultValue = "false",
```

Comment:
I think it would be convenient to the user for us to add `, specifiedDefaultValue = "true" here.  That would make it so that one could simply enter `export data --parallel` to get the behavior of `export data --parallel=true`.

Reply:
also, since there is a unspecified default value (false), is it really necessary to make it a mandatory option if "dir" is specified? can we just make it default to false(or true, whatever sensible) if user didn't add --parallel, but specified --dir?